<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usemon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="cpu">
      <item>
       <layout class="QVBoxLayout" name="cpuSumLayout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="cpuLayout">
        <item>
         <layout class="QHBoxLayout" name="cpuByCores">
          <item>
           <layout class="QVBoxLayout" name="cpuOdd"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="cpuEven"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="ramcpufreq">
      <item>
       <layout class="QVBoxLayout" name="ramLayout"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="cpufreq">
        <item>
         <layout class="QVBoxLayout" name="cpuFreqEven"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="cpuFreqOdd"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
